src/mesido/esdl/esdl_mixin.py

```diff
@@ -431,7 +431,8 @@ def esdl_heat_model_options(self) -> Dict:
         energy_system_options = self.energy_system_options()
         v_nominal = energy_system_options["estimated_velocity"]
         v_max = self.heat_network_settings["maximum_velocity"]
-        return dict(v_nominal=v_nominal, v_max=v_max)
+        v_max_gas = self.gas_network_settings["maximum_velocity"]
```

Comment:
Was this not already in another PR?

Reply:
I thought so too. Lets check when I merge main back in again.